### PR TITLE
Nagios module: minor change, added "start" parameter

### PR DESCRIPTION
**Pull request by dramaley**
##### SUMMARY
Add a "start" parameter to the nagios module, allowing the user to specify when a scheduled outage in Nagios begins.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
nagios.py

##### ADDITIONAL INFORMATION
This PR finishes the "start" parameter of the nagios module, allowing the user to set when a Nagios downtime should begin. The backend functions within the module already supported a start parameter; this change just exposes that. If the new parameter is not provided, it defaults to the current time. This preserves compability with existing playbooks.

### CODE REVIEW

**Comment by dramaley** [comment]
Closing and reopening to force CI, per the notes on the "needs_ci" and "stale_ci" labels.

**Comment by dramaley** [comment]
bot_status

**Comment by goneri** [comment]
I'm not sure what happened with the CI here, it seems to be running this time.

**Comment by dramaley** [comment]
Hello. This has been sitting here awhile. Is there any way to get a pull request looked at and accepted? Thank you!

**Comment by dramaley** [comment]
Closing and reopening to force CI, per the notes on the "stale_ci" label.

**Comment by dramaley** [comment]
Closing and reopening to force CI.

File: lib/ansible/modules/monitoring/nagios.py [hunk 1]
```diff
@@ -261,6 +273,7 @@ def main():
             comment=dict(default='Scheduling downtime'),
             host=dict(required=False, default=None),
             servicegroup=dict(required=False, default=None),
+            start=dict(default=int(time.time())),
```
Comment (goneri) [inline lib/ansible/modules/monitoring/nagios.py]: I think it's better to set `start` to `None` by default. In this case `_fmt_dt_str()` with initialize it to `now()` anyway. And this avoid the case where the same variable can be initialized at two different locations in the code.

**Review by goneri — CHANGES_REQUESTED** [review]
A little comment, otherwise the code looks good.

**Comment by goneri** [comment]
Thanks for the contribution @dramaley :-)

**Comment by dramaley** [comment]
Thank you for much @goneri for committing this!

**Pull request merged**